### PR TITLE
[MIG] partner_external_map: Migration to 16.0

### PR DESCRIPTION
**Pull request by imarulbala**

### CODE REVIEW

File: partner_external_map/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1,23 @@
+In the old days of Odoo/OpenERP, back in version 6.1, there was an official
```
Comment (legalsylvain) [inline partner_external_map/readme/DESCRIPTION.rst]: Is this text relevant for version 16 ? I think few people played with version 6.1..

Comment (rven) [reply partner_external_map/readme/DESCRIPTION.rst]: @legalsylvain maybe you can suggest what the text should be then?

Comment (legalsylvain) [reply partner_external_map/readme/DESCRIPTION.rst]: No idéa. I dont use it. Just describe what the module does ?

File: partner_external_map/README.rst [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+In the old days of Odoo/OpenERP, back in version 6.1, there was an official
+*google_map* module ; this module added a *Map* button on the partner form view
+and, when the user clicked on that button, it would open a new tab on its web
+browser and go to Google Map with a search on the address of the partner.
+
+This module aims at restoring this feature with several improvements:
```
Comment (StefanRijnhart) [inline partner_external_map/README.rst:33]:
```suggestion
This module adds buttons on the partner form to open a new tab with
the map or the route to the address of the partner.
```

File: partner_external_map/readme/DESCRIPTION.rst (hunk 1 above)
Comment (StefanRijnhart) [reply partner_external_map/readme/DESCRIPTION.rst]: I'm suggesting a replacement below.

**Comment by StefanRijnhart** [comment]
Can you amend your precommit commit? It's description is missing the module name:
image

**Review by StefanRijnhart — APPROVED** [review]

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**